Automattic/wp-calypso
# [Odie] New odie package

## Proposed Changes

It's been long time with some "junk" in Calypso, mainly for an experiment we did months ago, and our delays on "cleaning" the code. There are some changes that involves domains and emails, hence the reviewers added (automagically)

All the changes here are towards the creation of an "universal" package to be used anywhere. Within calypso, and not so far in time, out of it.

## Testing Instructions

The changes are involving changes in the plans sections, as well as domains and emails. Testing main functionality there is crucial. 

1. Change DNS settings
2. Add a new Mailbox
3. Remove a mailxbox
4. Go to plans and reach the bottom of the page via scroll.
5. Talk with Wapuu (it should be open for everybody, if not, add flags=wapuu to the url)

## Pre-merge Checklist
